lib/ui_components/linkable.rb

```diff
@@ -0,0 +1,66 @@
+module UiComponents
+  module Linkable
+    module InstanceMethods
+      attr_accessor :text, :icon
+
+      # additional anchor tag attributes (HTML5)
+      attr_accessor :href, :hreflang, :media, :rel, :target, :type
+
+      def tag(attributes = {})
+        {
+          href: nil,
+          hreflang: nil,
+          media: nil,
+          rel: nil,
+          target: nil,
+          type: nil
+        }.each_pair do |var, default|
+          send("#{var}=", attributes.fetch(var, default))
+        end
+        super
+      end
+
+      def tag_attributes
+        super.merge(
+        {
```

Comment:
Redundant curly braces around a hash parameter.<br>Indent the first parameter one step more than the previous line.